shielded-pool/src/event.rs

```diff
@@ -0,0 +1,13 @@
+use penumbra_chain::NoteSource;
+use penumbra_crypto::Nullifier;
+use tendermint::abci::{Event, EventAttributeIndexExt};
+
+pub fn spend(nullifier: Nullifier, source: NoteSource) -> Event {
+    Event::new(
+        "spend",
+        vec![
+            ("nullifier", nullifier.to_string()).index(),
+            ("source", source.to_string()).index(),
```

Comment:
I don't think we need to index the `source` here, since the event is already happening in the context of the transaction that `source` is pointing to. So we can just record the nullifier and omit the source.